From bbad01a1b0f7c3eeecad09a32098bf76c4c50ab5 Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Wed, 31 Jul 2019 18:01:16 +0700
Subject: [PATCH 025/433] ARM: shmobile: r8a77470: Add clock index macros for
 DT sources

Add some macros usable by device tree sources to reference r8a77470
 clocks by index.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 include/dt-bindings/clock/r8a77470-clock.h | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/include/dt-bindings/clock/r8a77470-clock.h b/include/dt-bindings/clock/r8a77470-clock.h
index a22af15..f70660b 100644
--- a/include/dt-bindings/clock/r8a77470-clock.h
+++ b/include/dt-bindings/clock/r8a77470-clock.h
@@ -19,15 +19,19 @@
 #define R8A77470_CLK_SDH		6
 #define R8A77470_CLK_SD0		7
 #define R8A77470_CLK_SD1		8
-#define R8A77470_CLK_RCAN		9
+#define R8A77470_CLK_Z			9
 
 /* MSTP0 */
 #define R8A77470_CLK_MSIOF0		0
+#define R8A77470_CLK_HUDI		18
+#define R8A77470_CLK_INTCRT		22
 
 /* MSTP1 */
 #define R8A77470_CLK_VCP0		1
 #define R8A77470_CLK_VPC0		3
 #define R8A77470_CLK_STB		4
+#define R8A77470_CLK_TSIF1		7
+#define R8A77470_CLK_TSIF0		8
 #define R8A77470_CLK_TMU1		11
 #define R8A77470_CLK_3DG		12
 #define R8A77470_CLK_2DDMAC		15
@@ -41,13 +45,16 @@
 /* MSTP2 */
 #define R8A77470_CLK_MSIOF2		5
 #define R8A77470_CLK_MSIOF1		8
+#define R8A77470_CLK_MFIS		13
 #define R8A77470_CLK_SYS_DMAC1		18
 #define R8A77470_CLK_SYS_DMAC0		19
+#define R8A77470_CLK_SEC_ENGIN		28
 
 /* MSTP3 */
 #define R8A77470_CLK_SDHI2		12
 #define R8A77470_CLK_SDHI1		13
 #define R8A77470_CLK_SDHI0		14
+#define R8A77470_CLK_PCIE		19
 #define R8A77470_CLK_USBHS_DMAC0_CH1	26
 #define R8A77470_CLK_USBHS_DMAC1_CH1	27
 #define R8A77470_CLK_CMT1		29
-- 
2.7.4

